### DISCUSSION TITLE
Functionality isn't working with TSML UI/WP 6.0.2/Mapbox combination

### DISCUSSION THREAD

**Discussion by mik3h0** [code4recovery/12-step-meeting-list#921] [Q&A] [answered]
I'm not sure what's going on here. When I enable TSML UI I do not get a "Request to change this listing" button that opens a new form. Instead I get a button that links directly to create an e-mail.

**Comment by kiyote33** [comment, marked as answer]
Hi @mik3h0 Mike,
The Feedback Enhancement add-on functionality is only available for our Legacy UI interface. The Feedback Enhancement feature is currently a work in progress for the TSML UI interface. Please be patient, as it is a feature that is coming down the road. If you want to review the enhanced feedback form, simply toggle the **User Interface Display** on our _Meetings/Import & Settings/Settings_ page. 

As this is an issue that comes up quite often, I will attempt to transfer it to the Discussion page instead of closing. 

Hmmm. get "Unable to convert this issue to a discussion. Category is not accessible to the actor"...
